README.md

```diff
@@ -87,21 +87,21 @@ $ rake module:push
 
 #### Configuring to push a module to the Puppet Forge
 
-Configure your credentials in `~/.puppetforge.yml`
+Configure your credentials in `~/.puppetforge.yml`. Forge API keys may be created and revoked from a user's
+profile on the [Forge website](https://forge.puppet.com/), where other profile details are managed.
 
 ```yaml
 ---
-username: myuser
-password: mypassword
+api_key: myAPIkey
 ```
 
-Or set the equivalent environment variables in your shell
+Or set the equivalent environment variable in your shell
 
 ```bash
-export BLACKSMITH_FORGE_USERNAME=myuser
```

Comment:
Is `BLACKSMITH_FORGE_USERNAME` still needed or can we also remove it from https://github.com/voxpupuli/gha-puppet/blob/v1/.github/workflows/release.yml#L49 ?

Reply:
We currently set a username and a password, but can't tell if it's required: 
https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.github/workflows/release.yml.erb#L21-L22

Reply:
When we drop oauth, you either an API key, token or basic auth (with username and password).

Now the question is: if oauth is dropped, should we still support a token for the Puppet forge?

Reply:
the username/password combo is still an option for Artifactory, I think